File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/content/GetFolderPathRequestModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.content;

public class GetFolderPathRequestModel {

	private String documentLocationRealName;
	private String folderId;
	
	public String getDocumentLocationRealName() {
		return documentLocationRealName;
	}
	public void setDocumentLocationRealName(String documentLocationRealName) {
		this.documentLocationRealName = documentLocationRealName;
	}
	public String getFolderId() {
		return folderId;
	}
	public void setFolderId(String folderId) {
		this.folderId = folderId;
	}
}

File: CloudFlow/FrontEnd-App/src/app/shared/model/get-folder-path-request.model.ts
import { JsonObject, JsonProperty } from "json2typescript";

@JsonObject
export class GetFolderPathRequestModel {

	public constructor(documentLocationRealName: string, folderId: string) {
		this.documentLocationRealName = documentLocationRealName;
		this.folderId = folderId;
	}

	@JsonProperty("documentLocationRealName", String)
	public documentLocationRealName: string;

	@JsonProperty("folderId", String)
	public folderId: string;
}